src/scss/components/_progress-indicator.scss

```diff
@@ -39,10 +47,8 @@
 
     // style layout
     @at-root .#{prefix('layout__row')} & {
-      @include margin(
-        calc(-1 * var(--of-progress-indicator-mobile-margin)),
-        $properties: (margin-left, margin-right)
-      );
+      @include margin(calc(-1 * var(--of-progress-indicator-mobile-margin)),
+        $properties: (margin-left, margin-right));
```

Comment:
this looks weird and the previous version was already what prettier outputs, so please restore.